docs/site/content/posts/08-build-deploy-apps-w-tce.md

```diff
@@ -0,0 +1,694 @@
+- Retrieving available packages...
+  NAME                                           DISPLAY-NAME        SHORT-DESCRIPTION
+  cert-manager.community.tanzu.vmware.com        cert-manager        Certificate management
+  contour.community.tanzu.vmware.com             contour             An ingress controller
+  external-dns.community.tanzu.vmware.com        external-dns        This package provides DNS synchronization functionality.
+  fluent-bit.community.tanzu.vmware.com          fluent-bit          Fluent Bit is a fast Log Processor and Forwarder
+  gatekeeper.community.tanzu.vmware.com          gatekeeper          policy management
+  grafana.community.tanzu.vmware.com             grafana             Visualization and analytics software
+  harbor.community.tanzu.vmware.com              harbor              OCI Registry
+  knative-serving.community.tanzu.vmware.com     knative-serving     Knative Serving builds on Kubernetes to support deploying and serving of applications and functions as serverless containers
+  kpack.community.tanzu.vmware.com               kpack               kpack builds application source code into OCI compliant images using Cloud Native Buildpacks
+  local-path-storage.community.tanzu.vmware.com  local-path-storage  This package provides local path node storage and primarily supports RWO AccessMode.
+  multus-cni.community.tanzu.vmware.com          multus-cni          This package provides the ability for enabling attaching multiple network interfaces to pods in Kubernetes
+  prometheus.community.tanzu.vmware.com          prometheus          A time series database for your metrics
+  velero.community.tanzu.vmware.com              velero              Disaster recovery capabilities
+  whereabouts.community.tanzu.vmware.com         whereabouts         A CNI IPAM plugin that assigns IP addresses cluster-wide
+```
+
+Things that stand out… Cert-Manager, Contour, Harbor, all of which I’ve already installed.
+
+Together we're going to install Knative Serving and kpack.
+
+## Installed packages
+
+In fact, we can check specifically to see which packages have been installed.
+
+```shell
+$ tanzu package installed list
+/ Retrieving installed packages...
+  NAME          PACKAGE-NAME                             PACKAGE-VERSION  STATUS
+  cert-manager  cert-manager.community.tanzu.vmware.com  1.6.1            Reconcile succeeded
+  contour       contour.community.tanzu.vmware.com       1.19.1           Reconcile succeeded
+  external-dns  external-dns.community.tanzu.vmware.com  0.8.0            Reconcile succeeded
+  harbor        harbor.community.tanzu.vmware.com        2.3.3            Reconcile succeeded
+```
+
+## Create project in Harbor (image registry)
+
+Since Harbor is already installed, we can go ahead and open the UI. We’ll just log in as admin, and let’s go ahead and create a new “demo” project to store the images we’re going to publish.
+
+!["Create a new demo project dashboard"](/img/deploy-app-post-img1.png)
```

Comment:
I believe this needs to be `../img` to correctly reference the image path.

Reply:
I'm confused, the images are showing up fine in the deploy preview to the site? :/

Reply:
Chatted in slack - I think this was just my confusion from other similar changes recently that were actually referencing between the versioned docs and the unversioned blogs. This looks like it's actually right, so please ignore me. :)